labs/accessing-optimizely-event-data/metadata.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Accessing Optimizely Enriched Event Data
+summary: This lab contains a handy CLI for exploring and downloading Optimizely Enriched Event Data.
+revisionDate: '2020-07-20'
+labels:
+  - data
+  - analysis
+  - odata
+  - enriched
+  - events
```

Comment:
nit: gut sense is that we should group `enriched events` since `enriched` on its own is not super descriptive

Reply:
Good point.  Fixed.